apps/bfd-db-migrator/src/main/assembly/zip.xml

```diff
@@ -1,30 +1,32 @@
-<?xml version="1.0"?>
-<assembly xmlns="http://maven.apache.org/plugins/maven-assembly-plugin/assembly/1.1.2" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/plugins/maven-assembly-plugin/assembly/1.1.2 http://maven.apache.org/xsd/assembly-1.1.2.xsd">
-  <id>zip</id>
-  <includeBaseDirectory>true</includeBaseDirectory>
-  <formats>
-    <format>zip</format>
-  </formats>
-  <fileSets>
-    <fileSet>
-      <directory>src/main/assembly/bin</directory>
-      <outputDirectory>/</outputDirectory>
-      <directoryMode>0755</directoryMode>
-      <fileMode>0755</fileMode>
-    </fileSet>
-  </fileSets>
-  <dependencySets>
-    <dependencySet>
-      <outputDirectory>/</outputDirectory>
-      <includes>
-        <include>${project.groupId}:${project.artifactId}:jar:*</include>
-      </includes>
-    </dependencySet>
-    <dependencySet>
-      <outputDirectory>lib</outputDirectory>
-      <excludes>
-        <exclude>${project.groupId}:${project.artifactId}:jar:*</exclude>
-      </excludes>
-    </dependencySet>
-  </dependencySets>
+<assembly xmlns="http://maven.apache.org/plugins/maven-assembly-plugin/assembly/1.1.2"
```

Comment:
On this `zip.xml`, it looks like this is just reformatting/tidying. Are you using a tool that produces this format, or was this lovingly created by hand?

Reply:
I just copy/pasted this from the one in pipeline; I assume when you did this originally your editor/IDE may have added some spacing/extra bits?